mautic/mautic
# Improve condition in lessphp lib

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Y]
| New feature/enhancement? (use the a.x branch)      | [N]
| Deprecations?                          | [N]
| BC breaks? (use the c.x branch)        | [N]
| Automated tests included?              | [N] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

So a while back there was this issue pointed out on the forums: https://forum.mautic.org/t/log-flooded-with-mautic-notice-php-notice-trying-to-access-array-offset-on-value-of-type-bool-redlining-server/19320/6?u=mzagmajster

and this commit fixes the library mentioned in the thread  in a way so we do not get php notice anymore.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a focus item.
3. Save the focus item.
4. There should be no notice in the log as its described in the link above




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10765"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
